apis/nucleus/src/components/Header.jsx

```diff
@@ -83,7 +83,13 @@ function Header({ layout, sn, anchorEl, hovering, focusHandler, titleStyles = {}
         <Grid container wrap="nowrap" direction="column">
           {showTitle && (
             <Tooltip title={layout.title}>
-              <Typography variant="h6" noWrap className={CellTitle.className} style={titleStyles.main}>
+              <Typography
```

Comment:
For classic chart, when there is no title will read no title. Probably better add something to make it work same.